document-models/real-world-assets/mock-data/make-initial-state.ts

```diff
@@ -0,0 +1,5 @@
+import { initialState } from './initial-state';
```

Comment:
this is just for convenience so i can use a strongly typed object and then copy paste it in the initial state json. its quicker for me than going through the app because I am already editing the files directly